extensions/spotify-player/package.json

```diff
@@ -30,6 +30,15 @@
     "andyburris",
     "thomaslombart"
   ],
+  "debug": {
+    "reloadShortcut": {
+      "key": "r",
+      "modifiers": [
+        "command",
+        "shift"
+      ]
+    }
+  },
```

Comment:
What is this used for?

Reply:
This changes the shortcut to reload the entire extension to cmd-shift-r to prevent conflicts, as cmd-r is already used in some places by the extension to . It's not documented in the [docs](https://developers.raycast.com), but it does appear in the [schema](https://www.raycast.com/schemas/extension.json).